File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;

        if (s.length() == 4 || s.length() == 6) {
            for (int i = 0; i < s.length(); i++) {
                char ch = s.charAt(i);

                if (!Character.isDigit(ch)) {
                    answer = false;
                    break;
                }
            }
        } else {
            answer = false; 
        }

        return answer;
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.js
function solution(s) {
    var answer = true;
    
    if(s.length !== 4 && s.length !== 6){
        answer = false;
    }
    
        for(let i of s){
        if(isNaN(i)){
            answer = false;
        }
    }
    
    return answer;
}